src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannRequestGenerator.cs

```diff
@@ -1067,17 +1067,15 @@ private IEnumerable<IndexerRequest> GetRequest(Dictionary<string, object> variab
                     }
                 }
 
-                var request = new CardigannRequest(requestbuilder.Build(), variables, searchPath);
-
                 // send HTTP request
-                if (search.Headers != null)
+                if (_definition.Search.Headers != null)
```

Comment:
Don't switch all these back, the search variable is set from the definition once at the beginning of the method so we don't have to call back to def field each go.